core/src/main/java/com/scalar/db/sql/Example.java

```diff
@@ -0,0 +1,488 @@
+package com.scalar.db.sql;
+
+import com.scalar.db.sql.builder.SqlBuilder;
+import com.scalar.db.sql.statement.BatchStatement;
+import com.scalar.db.sql.statement.CreateNamespaceStatement;
+import com.scalar.db.sql.statement.CreateTableStatement;
+import com.scalar.db.sql.statement.DeleteStatement;
+import com.scalar.db.sql.statement.DropNamespaceStatement;
+import com.scalar.db.sql.statement.DropTableStatement;
+import com.scalar.db.sql.statement.InsertStatement;
+import com.scalar.db.sql.statement.SelectStatement;
+import com.scalar.db.sql.statement.TruncateTableStatement;
+import com.scalar.db.sql.statement.UpdateStatement;
+import com.scalar.db.transaction.consensuscommit.Attribute;
+import com.scalar.db.transaction.consensuscommit.Coordinator;
+
+public class Example {
+
+  private static final String NAMESPACE_NAME = "ns";
+  private static final String TABLE_NAME = "tbl";
+  private static final String COLUMN_NAME_1 = "col1";
+  private static final String COLUMN_NAME_2 = "col2";
+  private static final String COLUMN_NAME_3 = "col3";
+  private static final String COLUMN_NAME_4 = "col4";
+
+  public static void main(String[] args) {
+    try (SqlSessionFactory sqlSessionFactory =
+        SqlSessionFactory.builder()
+            .withProperty("scalar.db.contact_points", "jdbc:mysql://localhost:3306/")
+            .withProperty("scalar.db.username", "root")
+            .withProperty("scalar.db.password", "mysql")
+            .withProperty("scalar.db.storage", "jdbc")
+            .build()) {
```

Comment:
We fist need to create this `SqlSessionFactory`. I'm going to add some more utility methods to the builder class of `SqlSessionFactory`.